test/jit/test_enum.py

```diff
@@ -175,10 +175,7 @@ class Color(Enum):
         make_global(Color)
 
         def enum_const(x: Color) -> bool:
-            if x == Color.PURPLE:
-                return True
-            else:
-                return False
+            return x == Color.PURPLE
```

Comment:
I would not touch jit tests, because may be some of those is validating that frontend parser can deal with those constructions